python/invoke_release/tasks.py

```diff
@@ -696,6 +696,22 @@ def _create_branch(verbose, branch_name):
     _verbose_output(verbose, 'Done creating branch {}.', branch_name)
 
 
+def _create_local_tracking_branch(verbose, branch_name):
+    _verbose_output(
+        verbose,
+        'Creating local branch {branch} set up to track remote branch {branch} from \'origin\'...',
+        branch=branch_name
+    )
+
+    subprocess.check_call(
+        ['git', 'checkout', '--track', 'origin/{}'.format(branch_name)],
```

Comment:
Why the `--track origin/branch_name`, etc.? I always just `git checkout branch_name` and it automatically sets up the local branch to track the remote. Are you just trying to be explicit, or is there a specific use case I'm missing here?

Reply:
Mostly trying to be explicit, but also to avoid unexpected behavior if people have atypical git configurations.
Per:
https://stackoverflow.com/questions/10002239/difference-between-git-checkout-track-origin-branch-and-git-checkout-b-branch/29784240#29784240
`git checkout branch`
is equivalent to
`git checkout --track origin/branch`
but can fail if branch exists on multiple remotes.

Reply:
👍